sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StatsBeat.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Metrics;
+using System.Threading;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal sealed class Statsbeat : IDisposable
+    {
+        private readonly IDisposable _meterProvider;
+
+        private static readonly Meter s_myMeter = new("AttachStatsBeatMeter", "1.0");
+
+        private const string ConnectionString = "<ConnectionString>";
+
+        private static string s_roleName;
+
+        private static string s_roleInstance;
+
+        private static string s_ikey;
+
+        internal static string s_os = StorageHelper.IsWindowsOS() ? "windows" : "linux";
```

Comment:
Seems odd to me that this comes from the StorageHelper.
We might want to consider moving this to a different class (RuntimeHelper ?).

The `RuntimeInformation.IsOSPlatform` actually supports Windows, Linux, and OSX. 
Does Statsbeat want to record OSX here?